package.json

```diff
@@ -29,19 +29,18 @@
   "license": "MPL-2.0",
   "dependencies": {
     "base64url": "2.0.0",
-    "bluebird": "2.1.3",
-    "clone": "1.0.2",
-    "convict": "1.4.0",
+    "bluebird": "3.5.0",
```

Comment:
I've made the changes for the upgrade to bluebird in this commit: change `done` to `then`, and update a `promisfy` call.